Bottom: 7e6828d9a4dbe3608aff84244997f42594963890
Top:    219243c4faed4f2a67ac8efd645667d73acd330b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 01:58:15 +0000

Refresh of remove-cygwin_cflags-and-cygwi

---

diff --git a/configure.ac b/configure.ac
index 1af732e..9c1b875 100644
--- a/configure.ac
+++ b/configure.ac
@@ -88,11 +88,6 @@ PCB_PATH_DELIMETER=":"
 PCB_DIR_SEPARATOR_S="/"
 PCB_DIR_SEPARATOR_C='/'
 case $host in
-	*-*-cygwin* )
-		CFLAGS="$CFLAGS ${CYGWIN_CFLAGS}"
-		CPPFLAGS="$CPPFLAGS ${CYGWIN_CPPFLAGS}"
-		;;
-
 	*-*-mingw* )
 		WIN32=yes
 		CFLAGS="$CFLAGS ${MINGW_CFLAGS:--mms-bitfields -mwindows}"
